iamproperty/iamproperty.github.io
# Ds 124/add guidelines

## Summary of Changes:
1. Add brand guidelines pdf

## Rationale
Increase visibility of the brand guidelines

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to the footer of the page. Then click on brand guidelines which then should load a pdf in a new page.

## Ticket(s)
DS-124

## Checklist

## Notes
